scripts/release/pypi/build.sh

```diff
@@ -20,8 +21,8 @@ pip install -U pip setuptools wheel
 pip list
 
 script_dir=`cd $(dirname $BASH_SOURCE[0]); pwd`
-echo $script_dir
-if [[ "$branch" != "release" ]]; then
+
+if [[ "$branch" == "dev" ]]; then
```

Comment:
why we have this change?

Reply:
For `beta` branch, we will manually modify the version to `3.0.0bN`, so we don't want to modify its version here either. For Python wheel edge builds, we actually only have the simple repo for `dev` branch now, so it's safe to make this change.